website/content/en/docs/cli/operator-sdk_create_webhook.md

```diff
@@ -40,7 +40,6 @@ operator-sdk create webhook [flags]
       --plural string             resource irregular plural form
       --programmatic-validation   if set, scaffold the validating webhook
       --version string            resource Version
-      --webhook-version string    version of {Mutating,Validating}WebhookConfigurations to scaffold. Options: [v1, v1beta1] (default "v1")
```

Comment:
The flag was marked as deprecated and because of this the generate doc is removing when ihmo should not

Reply:
But we do remove deprecated flags from the docs, right? Do we still need to have this since officially go/v3 still supports v1beta1?

Reply:
The docs are generated automatically (see: https://github.com/operator-framework/operator-sdk/blob/master/Makefile#L42). Then, we need to change/fix that. See: #5286